src/util/list.c

```diff
@@ -205,6 +227,50 @@ void remove_node(list_t *list, int key)
 
         free(temp);
     }
+    list->current = list->head;
+}
+
+void remove_nth_node(list_t *list, int n)
```

Comment:
Duplicates `remove_node()` functionality.

Reply:
But `remove_node` is key-based and I've already discussed the problem with using keys.